File: SCC0600/exercíciosPy/ex2.py
import math

# ENTRADA:
a = float(input())
b = float(input())
c = float(input())
res = None

# PROCESSAMENTO:
delta = b*b - 4 * a * c

if delta < 0:
  res = math.nan
else:
  x1 = (-b + math.sqrt(delta)) / (2 * a)
  x2 = (-b - math.sqrt(delta)) / (2 * a)
  res = x1 + x2

# SAIDA:
print("{:.2f}".format(res))

File: SCC0600/exerciciosC/ex2.c
#include <stdio.h>
#include <math.h>

int main(void) {
    float a; float b; float c;
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &c);

    float delta = (b * b) - (4 * a * c);

    if (delta >= 0) {
        float x1 = (-b + sqrt(delta)) / (2 * a);
        float x2 = (-b - sqrt(delta)) / (2 * a);

        float soma = x1 + x2;

        printf("%.2f\n", soma);
    } else {
        printf("%s\n", "nan");
    }
}